### PR TITLE
test for AIRBYTE_ENTRYPOINT env variable in connectors

### PR DESCRIPTION
**Pull request by jrhizor**
Final item required for closing https://github.com/airbytehq/airbyte/issues/3903

Tests for setting the entrypoint env variable in the source and destination standard tests.

I thought about trying to match this to the actual entrypoint for the image, but that seemed difficult since you'd need to combine parts of an array of strings, handle spacing correctly, etc (and even then the var doesn't technically _have_ to match the main entrypoint anyways).

### CODE REVIEW

File: airbyte-workers/src/main/java/io/airbyte/workers/process/KubePodProcess.java [hunk 1]
```diff
@@ -167,7 +167,7 @@ public static String getCommandFromImage(KubernetesClient client, String imageNa
           "Missing AIRBYTE_ENTRYPOINT from command fetcher logs. This should not happen. Check the echo command has not been changed.");
     }
 
-    var envVal = logs.split("=")[1].strip();
+    var envVal = logs.split("=", 2)[1].strip();
```
Comment (jrhizor) [inline airbyte-workers/src/main/java/io/airbyte/workers/process/KubePodProcess.java:170]: Allows for entrypoints that contain equals signs...

File: airbyte-integrations/bases/standard-source-test/src/main/java/io/airbyte/integrations/standardtest/source/SourceAcceptanceTest.java [hunk 2]
```diff
@@ -292,6 +292,16 @@ public void testEmptyStateIncrementalIdenticalToFullRefresh() throws Exception {
     assertSameRecords(fullRefreshRecords, emptyStateRecords, assertionMessage);
   }
 
+  /**
+   * In order to launch a source on Kubernetes in a pod, we need to be able to wrap the entrypoint.
+   * The source connector must specify its entrypoint in the AIRBYTE_ENTRYPOINT variable. This test
+   * ensures that the entrypoint environment variable is set.
+   */
+  @Test
+  public void testEntrypointEnvVar() throws Exception {
```
Comment (sherifnada) [inline airbyte-integrations/bases/standard-source-test/src/main/java/io/airbyte/integrations/standardtest/source/SourceAcceptanceTest.java:301]: this should also live in the [python source acceptance test suite](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_core.py)

Do you feel comfortable making that change? Alternatively can offload to connectors teams

**Review by sherifnada — APPROVED** [review]

File: airbyte-integrations/bases/standard-source-test/src/main/java/io/airbyte/integrations/standardtest/source/SourceAcceptanceTest.java (hunk 2 above)
Comment (jrhizor) [reply airbyte-integrations/bases/standard-source-test/src/main/java/io/airbyte/integrations/standardtest/source/SourceAcceptanceTest.java:301]: I'd prefer to offload to connector teams. Should be a lot faster for them to implement.

https://github.com/airbytehq/airbyte/issues/3903 is the main issue. Can you create another ticket if necessary?

**Review by davinchia — APPROVED** [review]

**Pull request merged**